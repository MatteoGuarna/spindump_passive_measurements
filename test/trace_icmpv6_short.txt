Two short ICMP echo-echo response pairs using IPv6.
