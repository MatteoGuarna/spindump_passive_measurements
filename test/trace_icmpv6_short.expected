ICMP 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:2b0:1c1::198 959 at 1553417897245943 new starting packets 0 0 bytes 0 0
ICMP 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:2b0:1c1::198 959 at 1553417897279742 measurement up right 33799 packets 1 0 bytes 56 0 bandwidth 56 0
ICMP 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:2b0:1c1::198 959 at 1553417898282108 measurement up right 35588 packets 2 1 bytes 112 56 bandwidth 56 56
